model/ddl.go

```diff
@@ -53,6 +53,8 @@ const (
 	ActionCreateView                   ActionType = 21
 	ActionModifyTableCharsetAndCollate ActionType = 22
 	ActionTruncateTablePartition       ActionType = 23
+	ActionDropTableOrView              ActionType = 24
```

Comment:
Why we need ActionDropTableOrView?

Reply:
DropTable and DropView's common part would use this flag.